changes/serializers.py

```diff
@@ -71,3 +71,13 @@ def to_internal_value(self, data):
 
         return super(ReceiveWhatsAppEventSerializer, self).to_internal_value(
             data)
+
+
+class ReceiveWhatsAppSystemEventSerializer(serializers.Serializer):
+
+    class EventSerializer(serializers.Serializer):
+        type = serializers.CharField(required=True)
```

Comment:
I made this a `CharField` instead of a `ChoiceField` because new types can be added to Engage at any time.